demos/basic-v16-rsc/e2e/playwright/support/on-rails.js

```
@@ -0,0 +1,37 @@
+}
+
+const app = (name, options = {}) => appCommands({ name, options }).then((body) => body[0])
+const appScenario = (name, options = {}) => app('scenarios/' + name, options)
+const appEval = (code) => app('eval', code)
+const appFactories = (options) => app('factory_bot', options)
+
+const appVcrInsertCassette = async (cassette_name, options) => {
+  const context = await contextPromise;
+  if (!options) options = {};
+
+  Object.keys(options).forEach(key => options[key] === undefined ? delete options[key] : {});
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)